src/power_divergence.jl

```diff
@@ -214,6 +235,48 @@ end
 
 # Under regularity conditions, their asymptotic distributions are all the same (Drost 1989)
 # Chi-squared null approximation works best for lambda near 2/3
+"""
+    PowerDivergenceTest(x [,y] [, lambda] [,theta0] )
+
+Perform a Power Divergence Test.
+
+If `x` is a matrix with one row or column, or if `x` is a vector and `y` is not given, then
+a goodness-of-fit test is performed (``x`` is treated as a one-dimensional contingency
+table. The entries of `x` must be non-negative integers. In this case, the hypothesis
+tested is whether the population probabilities equal those in `theta0`, or are all equal if
+`theta0` is not given.
+
+If `x` is a matrix with at least two rows and columns, it is taken as a two-dimensional
+contigency table: the entries of `x` must be non-negative integers. Otherwise, `x` and
```

Comment:
"contingency". It seems that "the entries of `x` must be non-negative integers" applies all the time, and also to `y` when it is passed, so maybe better mention this at the end in a separate paragraph?